Registry.php

```diff
@@ -75,7 +76,7 @@ private function resetOrClearManager(string $managerName, string $serviceId): vo
 
         assert($manager instanceof EntityManagerInterface);
 
-        if (! $manager instanceof LazyLoadingInterface || $manager->isOpen()) {
+        if ((! $manager instanceof LazyLoadingInterface && ! $manager instanceof LazyGhostObjectInterface) || $manager->isOpen()) {
```

Comment:
feel free to suppress this by adding the LazyGhostObjectInterface here https://github.com/doctrine/DoctrineBundle/blob/2d8779dbf8035bc13ae88b7c8537cfd662a4d689/psalm.xml.dist#L43-L43